ts-packages/web/src/app/spaces/[id]/deliberation/_components/dropdown/drop-down-menu.tsx

```diff
@@ -0,0 +1,23 @@
+export default function DropdownMenu({
+  ondelete,
+}: {
+  onclose: () => void;
+  ondelete: () => void;
+}) {
+  return (
+    <div className="w-56 bg-neutral-800 text-white rounded-lg shadow-lg py-2 space-y-1">
+      <div className="px-4 py-2 hover:bg-neutral-700 cursor-pointer rounded-md text-white text-[14px] font-semibold">
+        See committee list
+      </div>
+      <div className="px-4 py-2 hover:bg-neutral-700 cursor-pointer rounded-md text-white text-[14px] font-semibold">
```

Comment:
Please refrain from using px as much as possible and use text-sm or rem-based styles.